src/mecca/reactor/io/fs_watcher.d

```diff
@@ -9,13 +9,12 @@ module mecca.reactor.io.fs_watcher;
 import mecca.log;
 import mecca.reactor;
 
-version(linux) {
-    import mecca.reactor.io.inotify;
-    public import mecca.reactor.io.inotify : WatchDescriptor;
-    public import core.sys.linux.sys.inotify;
-} else {
-    static assert(false, "Unsupported platform for file watching");
```

Comment:
This `static assert` was useful, why remove?

Reply:
Because it's not implemented yet for macOS. Alternatively I can exclude the file in the Dub file.

Reply:
I don't understand. Removing the static assert only makes the compile time error worse, with no other effects, no?

Reply:
No. The whole file is now versioned to only include Linux: https://github.com/weka-io/mecca/pull/12/files#diff-5a9fd23e689adb5dac296d053d37bc03R12.